python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -30,27 +29,27 @@ class ComfortType(str, Enum):
 
 class RollingResistance(BaseModel, extra=Extra.forbid):
     type: Literal["davis"]
-    A: confloat(ge=0)
-    B: confloat(ge=0)
-    C: confloat(ge=0)
+    A: Annotated[float, Field(ge=0)]
+    B: Annotated[float, Field(ge=0)]
+    C: Annotated[float, Field(ge=0)]
 
 
 class EffortCurve(BaseModel, extra=Extra.forbid):
-    speeds: conlist(confloat(ge=0), min_items=2) = Field(
+    speeds: Annotated[List[Annotated[float, Field(ge=0)]], Field(min_length=2)] = Field(
         description="Curves mapping speed (in m/s) to maximum traction (in newtons)"
```

Comment:
```suggestion
    speeds: List[PositiveFloat] = Field(
            min_length=2,
            description="Curves mapping speed (in m/s) to maximum traction (in newtons)"
```
please create a `PositiveFloat = Annotated[float, Field(ge=0)]` and reuse it as needed. it gets hard to read as is